packages/web/src/pages/index.vue

```diff
@@ -0,0 +1,21 @@
+<script lang="tsx">
+import { Component, Vue } from 'vue-property-decorator'
+import { CreateElement } from 'vue'
+
+@Component({})
+export default class HomePage extends Vue {
+  appName: string = 'WebOK'
+
+  render (h: CreateElement) {
+    return (
+      <h1 class='blue'>Welcome to {this.appName}!</h1>
```

Comment:
Use `class` instead of `className` because JSX for Vue is different from React: https://github.com/vuejs/babel-plugin-transform-vue-jsx#difference-from-react-jsx